### ISSUE TITLE
how to get MS-Celeb-1M data set

### ISSUE THREAD

**Issue by feifei9099** [ZhaoJ9014/face.evoLVe.PyTorch#68]
Does anyone have a download link to this dataset, or is there any good way to get this datase. I only need the MS-Celeb-1M Low-Shot part.

**Comment by ZhaoJ9014** [comment]
> Does anyone have a download link to this dataset, or is there any good way to get this datase. I only need the MS-Celeb-1M Low-Shot part.

Hi, you can find the download link of MS-Celeb-1M in Sec. Data Zoo of face.evoLVe.

**Comment by feifei9099** [comment]
Hi, thanks, but I have one more question. I found 2 data sets of MS-Celeb-1M  in Data Zoo (Clean / Align_112x112). I want to do a small sample study, and the MS-Celeb-1M lowshot learning data set is what I needed. Is this part of the data set contained here, and which one should I download？

**Comment by ZhaoJ9014** [comment]
> Hi, thanks, but I have one more question. I found 2 data sets of MS-Celeb-1M in Data Zoo (Clean / Align_112x112). I want to do a small sample study, and the MS-Celeb-1M lowshot learning data set is what I needed. Is this part of the data set contained here, and which one should I download？

Hi, for your case, please check this Baidu Drive: https://pan.baidu.com/s/1H8Ai7qJGmKJzGZJIW03A8w, pw: ppkr

**Comment by feifei9099** [comment]
Hi, Zhao, thank you very much for the data set you provided me. I have read your paper for detail, and I know that the Low-Shot data set has training sets and validation sets.

- Low-Shot training set contains two subsets: Base set and Novel set. Base set contains 20k identities with 50 to 70 images for each.  Novel set contains 1k identities with only 1 image for each.

- Low-Shot validation set comprises 5 images for each identity from the Base set and 20 images for each
identity from the Novel set.

The dataset you provided to me is a validation set. I would like to ask if there is a download link for the training set, or the way to get it.

**Comment by aplusc98** [comment]
Hey,

Zhao , could you please post the above link again , it has expired and I would like to download the Ms Celeb low shot dataset
